node/utils/constants.ts

```diff
@@ -9,3 +9,6 @@ export const ORDER_TO_RETURN_VALIDATON: Record<
   OUT_OF_MAX_DAYS: 'OUT_OF_MAX_DAYS',
   ORDER_NOT_INVOICED: 'ORDER_NOT_INVOICED',
 }
+
+export const OMS_RETURN_REQUEST = 'oms-return-request'
```

Comment:
Should we make it more specific, since we will have different templates for different parts of the process?